### DISCUSSION TITLE
Can ROCm or any AMD GPU get a smart access storage like feature

### DISCUSSION THREAD

**Discussion by shott92** [ROCm/ROCm#2442] [Ideas]
So as per the title really, loading datasets and training a lot of transfers from disk to ram to GPU are used.
Is there a way to use a similar process that modern games are using and directly access the datasets from the storage device?

**Comment by johnnynunez** [comment]
> So as per the title really, loading datasets and training a lot of transfers from disk to ram to GPU are used. Is there a way to use a similar process that modern games are using and directly access the datasets from the storage device?

in Nvidia https://developer.nvidia.com/magnum-io https://developer.nvidia.com/gpudirect-storage
https://developer.nvidia.com/dali

I thinkt that the best idea in the future is to compress like consoles, a hardware that allows compress and  decompress on the fly
